frontend/src/metabase/admin/tasks/components/Logs/Logs.unit.spec.tsx

```diff
@@ -97,6 +97,18 @@ describe("Logs", () => {
         expect(screen.getByText(errMsg)).toBeInTheDocument();
       });
     });
+
+    it("should stop polling on unmount", async () => {
+      fetchMock.get("path:/api/util/logs", [log]);
+      const { unmount } = render(<Logs />);
+      await waitFor(() => [screen.getByText(new RegExp(log.process_uuid))]);
```

Comment:
We want to ensure we assert conditions explicitly.

```suggestion
      await waitFor(() => {
        expect(
          screen.getByText(new RegExp(log.process_uuid)),
        ).toBeInTheDocument();
      });
```

This ensures we assert that the process log is in the document, like in other tests.

Reply:
Done!